TOPIC 0/2 - CSS/index.css

```diff
@@ -0,0 +1,39 @@
+html {
```

Comment:
What about this items?
_4. If the class attribute finishes with r (example header, footer), the background must be magenta.
5. If the class attribute contains an a (example nav) but do NOT finish with r, the background must be blue._